docs/concepts/workloads/pods/pod.md

```diff
@@ -150,7 +150,7 @@ Pod is exposed as a primitive in order to facilitate:
 * clean composition of Kubelet-level functionality with cluster-level functionality &mdash; Kubelet is effectively the "pod controller"
 * high-availability applications, which will expect pods to be replaced in advance of their termination and certainly in advance of deletion, such as in the case of planned evictions, image prefetching, or live pod migration [#3949](http://issue.k8s.io/3949)
 
-There is new first-class support for stateful pods with the [StatefulSet](/docs/concepts/abstractions/controllers/statefulsets/) controller (currently in beta). The feature was alpha in 1.4 and was called [PetSet](/docs/concepts/workloads/controllers/petset/). For prior versions of Kubernetes, best practice for having stateful pods is to create a replication controller with `replicas` equal to `1` and a corresponding service, see [this MySQL deployment example](/docs/tutorials/stateful-application/run-stateful-application/).
```

Comment:
We have to change `/docs/concepts/abstractions/controllers/statefulsets/` to `/docs/concepts/workloads/controllers/statefulset.md`.

The reason is most files of `/docs/concepts/abstractions/controllers/` have been removed: [link](https://github.com/kubernetes/kubernetes.github.io/tree/master/docs/concepts/abstractions/controllers)